Gruntfile.js

```diff
@@ -125,6 +125,7 @@ grunt.initConfig( {
 			"test/reporter-html/single-testid.html",
 			"test/reporter-urlparams.html",
 			"test/moduleId.html",
+            "test/onerror/onerror.html",
```

Comment:
Oops, need to tabify this line. I can't wait until we can get ESLint indent working...